### DISCUSSION TITLE
Feat: Labels for EDDIEHUB

### DISCUSSION THREAD

**Discussion by adityaraute** [EddieHubCommunity/support#3391] [Ideas]
I am envisioning one label for all EddieHub contributors, through which we can filter issues and PRs easily. This will help people who are willing to contribute to Open Source, but might not be following any fellow members.

[a comment by an automated account is not shown]

**Comment by naomi-lgbt** [comment]
Are you referring to a label we would all put on our personal repositories?

What's the envisioned use-case here? I could see it helping folks find projects run by fellow community members, but they'd still need to filter by `good first issue` or `help wanted` and whatnot.

**Comment by adityaraute** [comment]
Well all other labels will have to be mentioned, just like we use them generally. Only the addition of this particular label will help list all issues in one place as desired. That's my aim.

**Comment by Vyvy-vi** [comment]
But, I suppose we could have the #good-first-issue and #first-timers to refer to personal projects

**Comment by adityaraute** [comment]
Not all issues are for first-timers. What if someone wants to go very deep in a repo for contribution on something that is more complex?

**Comment by naomi-lgbt** [comment]
That's what the `help wanted` label is for :D

Both `help wanted` and `good first issue` are also specially parsed by GitHub.

[a comment by an automated account is not shown]

**Comment by Panquesito7** [comment]
Can someone remove the `no-issue-activity` label, please? Same in many other issues. Thanks.

**Comment by adityaraute** [comment]
Is there any progress on this issue?

**Comment by eddiejaoude** [comment]
Do you mean every issue in EddieHub to have a specific label? so they can be found easily on GitHub?

**Comment by suitupstan** [comment]
They mean a special issue for contributors that are a part of EddieHub, but that is a bit non-inclusive IMHO

**Comment by naomi-lgbt** [comment]
I've gone ahead and converted this issue into a discussion so we can further explore the idea and reach a consensus.
